README.md

```diff
@@ -18,7 +17,19 @@ To get the source code of our SDKs and samples via **git** just type:
     git clone https://github.com/Azure/azure-mobile-services.git
     cd ./azure-mobile-services/
 
-## Change log
+## SDk Documentation references
+
+#### Azure App Service Mobile Apps
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v3)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v2)
+
+#### Mobile Services
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v2)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v1)
+* [Android](http://azure.github.io/azure-mobile-services/android/v2)
+* [Javascript](http://azure.github.io/azure-mobile-services/javascript/v1)
+
+## SDK Change log
```

Comment:
I have a pending change in the Mobile Apps branch changelog that I still need to fix, which titles things as "Azure Mobile Apps". FYI that's the term we're using in our docs.

The big issue is that the master branch is for Mobile Services, while the MobileApps branch is for vNext. So it's kind of weird to talk about MobileApps while in the master branch. 

Does this readme need to cover both Apps and Services?  If so, then it's not really a readme about the SDK repo anymore.